frontend/src/components/CartItem.tsx

```diff
@@ -20,6 +20,14 @@ interface Props extends Input {
 }
 
 const CartItem = ({ item_id, quantity, updateTotal, removeItem, isCart }: Input) => {
+  const CartItemStyle = {
+    borderRadius: '30px',
+    background: 'rgba(255, 255, 255, 0.6)',
+    backdropFilter: 'blur(10px)',
+    padding: '2% 30px 2% 30px',
+    margin: '2% 0 2% 0',
+  };
```

Comment:
The CSS should store into `.css` file.
Btw, you can use scss if you want to split the css into multiple files.

Reply:
That's because we had a plan of switching using tailwind.
The idea is that by writing the styles inside js, it might be easier to swap to tailwind.
Also, the historical backend team who still care about frontend shity styling mentioned that "i cant switch css and js in my head".